# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1232678465 18000
#      Thu Jan 22 21:41:05 2009 -0500
# Node ID 4b6558abe67591c74d10f75e115609f45dbb654b
# Parent  ab61b47d343584a12e9a7be0014eee834daed91f
display.cc: remove extraneous output statement

diff --git a/src/display.cc b/src/display.cc
--- a/src/display.cc
+++ b/src/display.cc
@@ -78,18 +78,16 @@ display_info::init (void)
 
       CGSize sz_mm = CGDisplayScreenSize (display);
 
       CGFloat ht_mm = sz_mm.height;
       CGFloat wd_mm = sz_mm.width;
 
       rx = wd * 25.4 / wd_mm;
       ry = ht * 25.4 / ht_mm;
-
-      std::cerr << depth << " bit depth" << std::endl;
     }
   else
     warning ("no graphical display found");
 
 #elif defined (HAVE_X_WINDOWS)
 
   const char *display_name = getenv ("DISPLAY");
 
